frontend/src/components/input-form/input-form.jsx

```diff
@@ -245,21 +245,29 @@ class InputForm extends React.Component {
     }
     
     // Creates markup for buttons
+    let forgotPasswordButton = null
     const buttons = this.props.buttons.map((button) => {
       const disabled =  false;
-      
+      if(button.name === 'Forgot password?') {
```

Comment:
why are you hardcoidng a button here? The whole purpose of this component is that it generates buttons